src/components/dls/Modal/Modal.tsx

```diff
@@ -43,7 +70,7 @@ const Subtitle = ({ children }) => (
 const Footer = ({ children }) => <div className={styles.footer}>{children}</div>;
 type ActionProps = {
   children: React.ReactNode;
-  onClick?: () => void;
+  onClick?: (e) => void;
```

Comment:
Should we add the type of the event here?

Reply:
✅